superset-frontend/src/components/ListViewCard/index.tsx

```diff
@@ -169,13 +182,14 @@ function ListViewCard({
   imgPosition = 'top',
   cover,
 }: CardProps) {
+  const Link = url && linkComponent ? linkComponent : AnchorLink;
   return (
     <StyledCard
       data-test="styled-card"
       cover={
         cover || (
           <Cover>
-            <a href={url}>
+            <Link to={url!}>
```

Comment:
Maybe we can adda. test to make sure this renders as expected with/without the url?